README.md

```diff
@@ -27,9 +27,18 @@ To use this dbt package, you must have the Fivetran **SAP** (sap.com) the respec
 - bseg
```

Comment:
A few other notes on the README are as follows:
- We need to upgrade the dbt version badge to match the version in the package (greater than 1.3.0 not 1.0.0).
  - ![image](https://github.com/fivetran/dbt_sap/assets/74217849/a029f467-f023-45a6-9b21-264263e7fdf8)
- You include a section on [passthrough columns](https://github.com/fivetran/dbt_sap/tree/feature/0employee-and-text-reports#passing-through-additional-fields), but we don't actually allow passthrough columns. This should be removed or updated to include passthrough column variables.
- Additionally, the optional config section should be enclosed in a collapsable. Can you add the collapsable to this section.
- Can we highlight that these staging models are intended to replicate the extractor reports (provided in our transformation package) and do not apply any renaming to the fields. If we also have a link to these extractors for reference that would be great!
- Be sure to update the package installation section to point to a dbt hub version. The first release will still be v0.1.0

Reply:
- Version bumped
- Passthrough columns section removed
- Staging model descriptions updated
- Update the package installation section

Reply:
Thanks! A few other small things I noticed that should be updated in the README that I noticed after this last update:
- Don't forget to add the databricks dispatch in the same location that you have in the transform README.
- The https://github.com/fivetran/dbt_sap_source/blob/feature/employee-text-reports/README.md#change-the-source-table-references section does not seem to be included in the collapsable section
- In the change-the-source-table-references you reference SAP2 sources. What are these?

Reply:
- Databricks dispatch updated
- </details> moved to include section in collapsible
- Uhh, good question. I think that's from the oldest version of this model, not sure what that's referencing. Removed.